subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/results/ResultsStoreHelper.java

```diff
@@ -55,6 +55,10 @@ public static String determineChannel() {
         return System.getProperty(SYSPROP_PERFORMANCE_TEST_CHANNEL, "commits");
     }
 
+    public static boolean isHistoricalChannel() {
+        return "historical".equals(determineChannel());
```

Comment:
Isn't the channel `historical-<branch-name>`? See https://github.com/gradle/gradle/blob/b36ce94a9a2e345e2f9dac53f5771a2124fa358c/buildSrc/subprojects/plugins/src/main/kotlin/org/gradle/plugins/performance/PerformanceTestPlugin.kt#L360

Reply:
Yes, I vaguely remember this rule but didn't find it. 

Changed to ` determineChannel().startsWith("historical-");`